Lib/bdb.py

```diff
@@ -17,6 +18,13 @@ class Bdb:
     This class takes care of details of the trace facility;
     a derived class should implement user interaction.
     The standard debugger class (pdb.Pdb) is an example.
+
+    The optional skip argument must be an iterable of glob-style
+    module name patterns.  The debugger will not step into frames
+    that originate in a module that matches one of these patterns.
+    Whether a frame is considered to originate in a certain module
+    is determined by the __name__ in the frame globals.
+
```

Comment:
Thanks!  I wasn't sure on that one.  PEP-8 doesn't have it, but existing strings in this module did.  Probably a good time to remove them all.